lib/ajax_requests/ajax.js

```diff
@@ -27,4 +27,35 @@ module.exports = class Ajax{
       url: (`${apiURL}/api/v1/meals/${meal}/foods/${food}`)
     })
   }
+
+  static getAllFoods(){
+    return $.getJSON(`${apiURL}/api/v1/foods`)
+  }
+
+  static postFoods(name, calories){
+    const params = { food: { name: name, calories: calories} }
+    return $.post(`${apiURL}/api/v1/foods`, params)
+    .catch(errorLog)
+  }
+
+  static updateFood(food, id){
+    $.ajax({
```

Comment:
this ajax format is so much cleaner than the way i do it.  especially with the success messages!